.github/workflows/build.yml

```diff
@@ -116,6 +116,7 @@ jobs:
           -Dorg.gradle.workers.max=1
           -Pandroid.testoptions.manageddevices.emulator.gpu="swiftshader_indirect"
           -Pandroid.experimental.testOptions.managedDevices.emulator.showKernelLogging=true
+          -Dorg.gradle.jvmargs=-Xmx2048m
```

Comment:
I don't mind, but could also be a separate PR with a reasoning in the description.